### PR TITLE
Dbpedia check

### PR DESCRIPTION
**Pull request by lintangsutawika**

### CODE REVIEW

**Review by awebson — CHANGES_REQUESTED** [review]
This was not your but the original prompt author's fault: Could you please rewrite the templates to be more natural English? "Refer to" reads awkwardly with "what category" or "topic". Maybe say something like "this paragraph talks about a company", "the topic of this passage is an artist". Something like that. Thanks!

**Comment by lintangsutawika** [comment]
No problem. Will change now.

File: promptsource/templates/dbpedia_14/templates.yaml [hunk 1]
```diff
@@ -74,15 +46,16 @@ templates:
     - Written Work
     answer_choices_key: null
     id: 824ecb55-ecad-40c2-8033-f2fa0add2ddf
-    jinja: '{{title}} - {{content}} The text refers to what category? ||| {{ answer_choices[label]
+    jinja: '{{title}} - {{content}} What does the paragraph talk about? ||| {{ answer_choices[label]
```
Comment (awebson) [inline promptsource/templates/dbpedia_14/templates.yaml:49]: Hmm… Wording it in an interrogative sentence like this still reads weird. Like a normal human is expected to answer the entity's name, not a category name. A declarative sentence like what I suggested earlier is better, but that'd require adding determiner agreement "a company" vs. "an athlete" in the targets. Let's just delete this template for now.

**Review by awebson — CHANGES_REQUESTED** [review]
See my comment below. These answer choices just read weird as answers to the questions as they're phrased. Let's remove all templates that don't have choices_in_prompt for now. Alternatively, you can rewrite some templates to include answer choices in their prompts. Don't feel obligated to have 5 original task templates for now.

File: promptsource/templates/dbpedia_14/templates.yaml (hunk 1 above)
Comment (lintangsutawika) [reply promptsource/templates/dbpedia_14/templates.yaml:49]: What if I add the indefinite article in the answer choices?

**Review by awebson — APPROVED** [review]
Thank you!

**Pull request merged**